terraform/azure-ghaf-infra.tf

```diff
@@ -43,4 +64,62 @@ resource "azurerm_network_security_group" "ghafhydra_terraform_nsg" {
     destination_address_prefix = "*"
   }
 }
-
+# Create Storage Account
+resource "azurerm_storage_account" "ghafinfra_tf_storage_account" {
+  name                     = "ghafinfrastorage"
+  location                 = var.resource_group_location
+  resource_group_name      = azurerm_resource_group.ghaf_infra_tf_dev.name
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
```

Comment:
Isn't the storage account already created in https://github.com/tiiuae/ghaf-infra/blob/766b4fb424668cdc64c90c88101fcb72516f4114/terraform/azure-storage/tfstate-storage.tf#L32 ?

Reply:
I think we should completely remove the creation of the storage account resource here.